.readthedocs.yaml

```diff
@@ -0,0 +1,30 @@
+version: 2
+
+build:
+  os: ubuntu-20.04
+  apt_packages:
+    - graphviz
+  tools:
+    python: "3.9"
```

Comment:
Might as well use 3.10 or 3.11 to start out, get ahead of future deprecations.

Reply:
Good idea! Done in https://github.com/spacetelescope/slitlessutils/pull/25/commits/037317a9904b5afe2ce3b13cc4d9a2301678404f